### PR TITLE
Scrum 2765

### PR DESCRIPTION
**Pull request by ianlongden**

### CODE REVIEW

File: agr_literature_service/lit_processing/oneoff_scripts/make_subset_db/make_subset.py [hunk 1]
```diff
@@ -14,24 +14,102 @@
...
+    'AGRKB:101000000872479',
+    'AGRKB:101000000865324']
+
 ALL_OUTPUT = 2  # For verbosity
 helptext = r"""
     env PSQL_XXX used for Target subset db details.
     env ORIG_XXX used for source reference db details.
     If ORIG_XXX is defined use that for source db details.
-    If ORIG_XXX NOT defined then use PSQL_XXX details.
+    If ORIG_XXX NOT defined then PSQL_XXX details wil be used.
```
Comment (azurebrd) [inline agr_literature_service/lit_processing/oneoff_scripts/make_subset_db/make_subset.py:35]: typo "will be used" but not important

File: agr_literature_service/lit_processing/oneoff_scripts/make_subset_db/make_subset.py [hunk 2]
```diff
@@ -14,24 +14,102 @@
...
     In most cases ORIG_XXX will not be needed but this is incase
     the target and source databases are not on the same server.
 
 """
 parser = argparse.ArgumentParser(formatter_class=argparse.RawDescriptionHelpFormatter, description=helptext)
 parser.add_argument('-l', '--limit', help='Limit to x references', type=int, default=2000, required=False)
 parser.add_argument('-v', '--verbose', help="verbosity, minumum=0, headlines=1, full=2,", type=int, default=1, required=False)
+parser.add_argument('-d', '--dump_dir', help='Directory to dump schemas and dbs too', type=str, required=True)
```
Comment (azurebrd) [inline agr_literature_service/lit_processing/oneoff_scripts/make_subset_db/make_subset.py:43]: probably typo "and dbs to", but maybe not

File: agr_literature_service/lit_processing/oneoff_scripts/make_subset_db/make_subset.py [hunk 3]
```diff
@@ -14,24 +14,102 @@
...
+parser.add_argument('-f', '--full_orig_dump', help='Dump the original database in full aswell', type=bool, default=False, required=False)
```
Comment (azurebrd) [inline agr_literature_service/lit_processing/oneoff_scripts/make_subset_db/make_subset.py:44]: even smaller typo "as well"

**Review by azurebrd — DISMISSED** [review]
Thanks @ianlongden .  Great to see how to alter the triggers to disable / enable, for future large loads / updates.  I'm kinda wondering if the make_subset.py should be in oneoff_scripts/ or somewhere else, since they'll keep getting used, while in theory the oneoff_scripts/ might get purged at some point ?  I'd defer to @valearna and @sweng66 on the organization of the repo.

File: agr_literature_service/lit_processing/oneoff_scripts/make_subset_db/make_subset.py (hunk 1 above)
Comment (ianlongden) [reply agr_literature_service/lit_processing/oneoff_scripts/make_subset_db/make_subset.py]: fixed. Thanks

File: agr_literature_service/lit_processing/oneoff_scripts/make_subset_db/make_subset.py (hunk 2 above)
Comment (ianlongden) [reply agr_literature_service/lit_processing/oneoff_scripts/make_subset_db/make_subset.py]: fixed. Thanks

File: agr_literature_service/lit_processing/oneoff_scripts/make_subset_db/make_subset.py (hunk 3 above)
Comment (ianlongden) [reply agr_literature_service/lit_processing/oneoff_scripts/make_subset_db/make_subset.py]: fixed. Thanks

**Review by azurebrd — DISMISSED** [review]

**Comment by ianlongden** [comment]
I have updated this a fair bit since the last commit as I had time this morning. Mainly code repetition removal and added resources for the test references. This may need another quick review.

**Review by azurebrd — APPROVED** [review]
Cool !

**Pull request merged**